landscape/client/user/management.py

```diff
@@ -313,13 +328,25 @@ def add_user(self, message):
                 email,
                 sudoer=sudoer,
                 force_managed=force_managed,
+                known=known,
             )
         except SnapdHttpException as e:
             result = json.loads(e.args[0])["result"]
             raise UserManagementError(result)
 
         return response.result
 
+    def _add_system_user_assertion(self, assertion):
+        """Add a system user assertion."""
+        try:
+            # adding an assertion is idempotent
+            snap_http.add_assertion(assertion)
+        except SnapdHttpException as e:
+            result = json.loads(e.args[0])["result"]
```

Comment:
In view or https://github.com/Perfect5th/snap-http/pull/8 being merged can we perhaps use `SnapdHttpException.json` here instead?

Reply:
I've made the changes